src/main/php/predicate/ArraySubset.php

```diff
@@ -0,0 +1,90 @@
+<?php
+declare(strict_types=1);
+/**
+ * This file is part of bovigo\assert.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace bovigo\assert\predicate;
+
+use SebastianBergmann\Comparator\ComparisonFailure;
+use SebastianBergmann\Exporter\Exporter;
+
+class ArraySubset extends Predicate
```

Comment:
There should be a doc comment with a short explanation what the class does. Also, a `@since 6.2.0` tag for completeness.